app/src/main/java/ca/bc/gov/bchealth/datasource/DataStoreRepo.kt

```diff
@@ -1,69 +0,0 @@
-package ca.bc.gov.bchealth.datasource
-
-import android.content.Context
-import androidx.datastore.preferences.core.booleanPreferencesKey
-import androidx.datastore.preferences.core.edit
-import androidx.datastore.preferences.core.stringPreferencesKey
-import androidx.datastore.preferences.preferencesDataStore
-import ca.bc.gov.bchealth.BuildConfig
-import javax.inject.Inject
-import kotlinx.coroutines.flow.Flow
-import kotlinx.coroutines.flow.map
-
-/**
- * [DataStoreRepo]
- *
- * @author amit metri
- */
-private val Context.dataStore by preferencesDataStore(BuildConfig.APPLICATION_ID + "_preferences")
-
-class DataStoreRepo @Inject constructor(
```

Comment:
This is really not a recommended practice.
You don't require to delete this class and create a new class.
Instead, you need to pass encrypted shared preference in the class and modify its method as required.

This will avoid all the changes you did in the view models file for importing package and variable names.

Reply:
Changes are done on https://github.com/bcgov/myhealthBC-android/pull/15.  Please reject this PR as I have to revert the commits to restore the deleted files.